src/features/dashboard/components/AppForm/index.tsx

```diff
@@ -23,7 +23,7 @@ type TAppFormProps = {
   isUpdating?: boolean;
   submit: (data: IRegisterAppForm) => void;
   is_update_mode?: boolean;
-  form_is_cleared?: boolean;
+  formIsCleared?: boolean;
```

Comment:
why it is optional here. in above refs where its been called each component is sending `formIsCleared` property so we don't need to make it optional.